packages/process/process-monitor.pony

```diff
@@ -95,28 +95,28 @@ Document waitpid behaviour (stops world)
 """
 use "files"
 use @pony_os_errno[I32]()
-use @pony_asio_event_create[AsioEventID](owner: AsioEventNotify, fd: U32, 
+use @pony_asio_event_create[AsioEventID](owner: AsioEventNotify, fd: U32,
       flags: U32, nsec: U64, noisy: Bool)
 use @pony_asio_event_unsubscribe[None](event: AsioEventID)
 use @pony_asio_event_destroy[None](event: AsioEventID)
-      
-primitive _EINTR          fun apply(): I32 => 4
-primitive _STDINFILENO    fun apply(): U32 => 0
-primitive _STDOUTFILENO   fun apply(): U32 => 1
-primitive _STDERRFILENO   fun apply(): U32 => 2
-primitive _FSETFL         fun apply(): I32 => 4
-primitive _FGETFL         fun apply(): I32 => 3
-primitive _FSETFD         fun apply(): I32 => 2
-primitive _FGETFD         fun apply(): I32 => 1
-primitive _FDCLOEXEC      fun apply(): I32 => 1
-primitive _SIGTERM        fun apply(): I32 => 15
-
-primitive _EAGAIN      fun apply(): I32 =>
+
+primitive _EINTR fun apply(): I32 => 4
+primitive _STDINFILENO fun apply(): U32 => 0
+primitive _STDOUTFILENO fun apply(): U32 => 1
+primitive _STDERRFILENO fun apply(): U32 => 2
+primitive _FSETFL fun apply(): I32 => 4
+primitive _FGETFL fun apply(): I32 => 3
+primitive _FSETFD fun apply(): I32 => 2
+primitive _FGETFD fun apply(): I32 => 1
+primitive _FDCLOEXEC fun apply(): I32 => 1
+primitive _SIGTERM fun apply(): I32 => 15
+
+primitive _EAGAIN fun apply(): I32 =>
```

Comment:
I still think the `fun` should be on a separate line, since the numbers can't be aligned with the others.